ca.desrt.dconf-editor.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "ca.desrt.dconf-editor",
+    "base-version": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "dconf-editor",
+    "rename-icon": "dconf-editor",
+    "copy-icon": true,
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--socket=wayland",
+	    "--talk-name=ca.desrt.dconf"
```

Comment:
To be clear to reviewers from the future. dconf-editor works so long as it can see *something* without all of these it couldn't see everything.